deployment/updates/scripts/xml/notifications/2024_01_18_batchJobEmailNotificationFailure.xml

```diff
@@ -43,10 +43,22 @@
               <code>$scope->getEvent()->getBatchJob()->getStatus() == BatchJob::BATCHJOB_STATUS_FAILED</code>
             </field>
           </item>
+          <item objectType="KalturaEventFieldCondition">
+            <description>Object is asset</description>
+            <field objectType="KalturaEvalBooleanField">
+              <code>!is_null(assetPeer::retrieveById($scope->getEvent()->getBatchJob()->getObjectId()))</code>
+            </field>
+          </item>
+          <item objectType="KalturaEventFieldCondition">
+            <description>Object is caption asset</description>
+            <field objectType="KalturaEvalBooleanField">
+              <code>(assetPeer::retrieveById($scope->getEvent()->getBatchJob()->getObjectId()))->getType() == CaptionPlugin::getAssetTypeCoreValue(CaptionAssetType::CAPTION)</code>
+            </field>
+          </item>
         </eventConditions>
         <format>1</format>
-        <subject>[Kaltura] - Http notification failed</subject>
-        <body><![CDATA[ <p>Entry with ID {entry_id} failed with HTTP notification</p> entry name:{entry_name}  - PID:{pid} - ObjectId: {object_id} <p>Message:{event_notification_msg}</p> ]]></body>
+        <subject>[Kaltura] - Http notification for captions has failed</subject>
+        <body><![CDATA[ <p>Entry with ID {entry_id} failed with HTTP notification</p> entry name:{entry_name}  - PID:{pid} - CaptionAssetId: {object_id} <p>Message:{event_notification_msg}</p> ]]></body>
```

Comment:
Replace body text to:
HTTP notification failed for PID {pid} entry ID {entry_id} entry name {entry_name} caption asset ID {object_id} <p>Message:{event_notification_msg}

Reply:
Done.